manifests/init.pp

```diff
@@ -119,27 +119,39 @@
         }
         default:{
           case $version {
-            /^1\.8.*$/:{
+            /^1\.8(\..*)*$/:{
```

Comment:
since we don't use those match groups somewhere else we could simply leave them out

suggested: `/^1\.8\..*$/`